test/passes/simplify-globals_all-features.txt

```diff
@@ -230,3 +226,10 @@
   )
  )
 )
+(module
+ (type $none_=>_none (func))
+ (global $write-only i32 (i32.const 1))
```

Comment:
Any idea why this global isn't completely eliminated?  or is that done by a different pass?

Reply:
I think it will be done by [`RemoveUnusedModuleElements`](https://github.com/WebAssembly/binaryen/blob/master/src/passes/RemoveUnusedModuleElements.cpp).

Reply:
And this test doesn't run that pass?  These test are just running a single pass?

Reply:
Yes. For tests in tests/passes, test file names are command line arguments passed to `wasm-opt`, `_` being a splitter between passes. So if a file name is `abc_def.txt`, this will run with `wasm-opt --abc --def`. So this file runs with `wasm-opt --simplify-globals --all-features`. If you want, you can add a test that runs with multiple passes by naming a file something like `simplify-globals_remove-unused-module-elements_all-features.txt`.

Reply:
Thanks!